core/controllers/story_viewer.py

```diff
@@ -39,6 +40,14 @@ class StoryPageDataHandler(base.BaseHandler):
     """
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'classroom_url_fragment': constants.SCHEMA_FOR_CLASSROOM_URL_FRAGMENTS,
+        'topic_url_fragment': constants.SCHEMA_FOR_TOPIC_URL_FRAGMENTS,
+        'story_url_fragment': constants.SCHEMA_FOR_STORY_URL_FRAGMENTS,
```

Comment:
Wait I see these params here but they are not used in the function below, do you know why is that?

Reply:
according to the wiki, we add all URL path elements into URL_PATH_ARGS_SCHEMAS whether these are used or not. you can check `PracticeSessionsPageDataHandler` for reference.

![Screenshot from 2021-10-05 15-01-40](https://user-images.githubusercontent.com/42772952/135997936-eaf238f4-815d-424b-9766-f3da431b30fb.png)

Reply:
Ah, okay. Good point.